app/javascript/components/Card.js

```diff
@@ -24,8 +24,8 @@ export const CardFace = (props) => {
 
   return (
     <Drag className={"card card-" + props.card.id} onClick={handleClick} dataItem={props.dragEventId} >
-      <div style={{display: 'none'}} className="card__type">Visible: {props.card.name}{props.card.pending ? ' pending' : ''}</div>
-      <div className={"card__element card__" + props.card.deck + "-deck card--size-" + props.size}>
+      <div style={{display: 'none'}} className="card__type">{props.title}: {props.card.name}{props.card.pending ? ' pending' : ''}</div>
+      <div title={props.title} className={"card__element card__" + props.card.deck + "-deck card--size-" + props.size}>
```

Comment:
you suck

Reply:
very constructive.... I felt this was a good change as it removes the need for a header, freeing up screen space, but still allows the user to check what stack it is by hovering over the card???